portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addMovieQuote(){
+    const quotes = ['May the force be with you', 'My Suits Are On The House, Or The House Burns Down.', 'Everyone is A Whore, Grace. We Just Sell Different Parts Of Ourselves', 'Show me the money','Already Broken.'];
```

Comment:
I'd take the "whore" line out, as eventually you'll be showing these to other people and it can be seen as offensive